testing/src/main/java/io/grpc/testing/TestUtils.java

```diff
@@ -217,11 +217,18 @@ public static File loadCert(String name) throws IOException {
    * Creates an SSLSocketFactory which contains {@code certChainFile} as its only root certificate.
    */
   public static SSLSocketFactory newSslSocketFactoryForCa(File certChainFile) throws Exception {
+    return newSslSocketFactoryForCa(new FileInputStream(certChainFile));
```

Comment:
I'm honestly reasonably okay with that as well. It looked easier to fix now. Either way; it was a pre-existing problem any way.